test/unit/unit.c

```diff
@@ -3,6 +3,20 @@
 #include <stdio.h>
 #include <stdarg.h>
 
+/*
+ * The definition of function set_sigint_cb is needed to avoid the error
+ * while linking. This occurs because libbox.a contains the unresolved
+ * lbox_console_readline() symbol, and it is necessary to construct a unit
+ * test executable.
+ */
+typedef struct ev_loop ev_loop;
+struct ev_signal;
+typedef void (*sigint_cb_t)(ev_loop *loop, struct ev_signal *w, int revents);
+sigint_cb_t
+set_sigint_cb(sigint_cb_t new_sigint_cb){
+	(void)new_sigint_cb;
+	return NULL;
+}
```

Comment:
It doesn't feel right that these box-specific stubs are defined in `unit.c`, which only contains unit test related stuff. Can you move it to a separate file, like `core_test_utils.c`?

Reply:
There is a duplicate symbol in `core_test_utils.c` which is '_cord_on_yield'.

Reply:
I don't understand you. I asked if you can move this mock `set_sigint_cb` function to a separate file (e.g. `box_test_utils.c`) so as not to pollute `unit.c`, like we did with `cord_on_yield`.

Reply:
Sorry, initially I understood you wrong.

Reply:
Fixed.